tripleblindmarket/private-kit
# Update Exposure Notification Status After EN is Enabled

#### Description:
Implements ENStatus check on native layer to update JS layer on the status of Exposure Notifications. 


#### Screenshots:


![Screenshot_20200703-114313](https://user-images.githubusercontent.com/3731917/86492337-aef43880-bd22-11ea-8ca8-285b2ed80866.png)


#### How to test:
Fresh install app.Go through onboarding and Enable Exposure Notifications. Observe that upon accepting EN, the UI updates right away to convey to the user that EN are enabled.
